src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/UploadFileItem.vue

```diff
@@ -8,24 +8,34 @@
           <p class="text-no-wrap">&nbsp;({{ Functions.humanFileSize(fileItem.file.size) }})</p>
         </div>
       </v-col>
+      <v-col cols="2">
+        <v-icon v-if="fileItem.fileErrors.length > 0" color="#CE3E39" icon="mdi-alert-circle"></v-icon>
+      </v-col>
 
       <!-- Progress Bar or Upload Completed -->
       <v-col cols="4">
-        <div v-if="isUploadComplete">Upload completed</div>
-        <v-progress-linear v-else :model-value="uploadProgress" height="20" color="primary"></v-progress-linear>
+        <div v-if="!(fileItem.fileErrors.length > 0)">
+          <div v-if="isUploadComplete">Upload complete</div>
+          <v-progress-linear v-else :model-value="uploadProgress" height="20" color="primary"></v-progress-linear>
+        </div>
       </v-col>
 
       <!-- Delete Button -->
-      <v-col cols="4" class="d-flex justify-end">
+      <v-col cols="2" class="d-flex justify-end">
         <v-tooltip text="Delete" location="top">
```

Comment:
tool tip looks different than figma with the blue box? 
Screenshot
![image](https://github.com/user-attachments/assets/0ed71104-d168-45eb-a516-dd1739cc9db7)
Figma 
![image](https://github.com/user-attachments/assets/4181552e-9bf2-464c-bab6-6351ae726800)

Ignore if it's okay. Code look similar to the other tooltips we have.

Reply:
Leaving this as it is for now to be consistent with other tooltips in the application. would adjust later if design flags it as a must have..